### ISSUE TITLE
Planned Analysis: Figure showing the distribution of samples by cancer type

### ISSUE THREAD

**Issue by cgreene** [AlexsLemonade/OpenPBTA-analysis#5]

**Comment by PichaiRaman** [comment]
This could include a graphic of a brain to show the origin / location of the different brain tumor types.

**Comment by cbethell** [comment]
To address the first part of this, would a bar plot with the cancer type on the x axis and the frequency on the y axis be well suited for this purpose? Possibly, a more complex version of this: 
![MSI-Fig1-860x685](https://user-images.githubusercontent.com/43576623/61975254-a0e85480-afb6-11e9-8e41-361efd0755d7.jpg)

or part A of this:
![1-s2 0-S2211124718304376-gr1](https://user-images.githubusercontent.com/43576623/61975554-734fdb00-afb7-11e9-908b-fc9c32bca6e5.jpg)

**Comment by cgreene** [comment]
@cbethell I think that the bar cart from the first paper would be aligned with what I was thinking here! I'm pretty sure the vision would be to use Column O in https://docs.google.com/spreadsheets/d/1Sa0hNX1lje40HdBpWiLUBY6-53UzvQczFLYb3uIm2jw/edit#gid=1663834900 .

**Comment by cbethell** [comment]
@cgreene  Great! Thank you for the clarification.

**Comment by jharenza** [comment]
@cbethell agree with @cgreene on the barplot and Column O!

**Comment by cbethell** [comment]
The graph below is a rough draft of my interpretation of the first figure suggested by @cgreene.

![image](https://user-images.githubusercontent.com/43576623/62245249-2d2bba80-b3af-11e9-8cab-53b10b99aa6b.png)

As is, the bars are colored by `count`, but this can later be changed to reflect `molecular_subtype`. The y-axis also represents the raw sample count, which can be changed to percentage if preferred. What are the thoughts on the above bar plot thus far?

**Comment by cgreene** [comment]
This is really helpful to me. It's nice to understand the characteristics of the data. @jharenza: does this align with what you were expecting the distributions to look like in the dataset?

**Comment by jharenza** [comment]
This looks right and the coloring in this case would be fine being all black or later colored by subtypes, but may make the legend complex. You can also see https://pedcbioportal.kidsfirstdrc.org/study/summary?id=pbta_cbttc as a guide for this dataset. I was also thinking a multi-venn could be useful here with inner rings being the broad histology, outer being these unique histologies, and next circle can be molecular subtype. Example is the html image [here](https://pecan.stjude.cloud/). I have some code for this (made in R) - will push to my last paper repo tomorrow and share. We could also consider assigning specific colors for each histology that we use throughout the paper for consistency. See some of the TCGA papers for color scheme examples.

**Comment by jharenza** [comment]
@cbethell I pushed the code here: https://github.com/marislab/create-pptc-pdx-pie - let me know if you have any issues!

**Comment by cbethell** [comment]
@jharenza Thank you! I'll take a look at it now.

**Comment by cbethell** [comment]
In addition to the [draft PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/40) I filed, I have put together a graphic of the brain showing the sites of various cancer types as suggested by @PichaiRaman.

Given the relatively large amount of unique cancer types in the dataset, I thought it may be more feasible to label the graphic with the highest expressed cancer type at each primary site. 

Let me know your thoughts on this concept (I'm sure the graphic itself can be further manipulated for better presentation so please feel free to provide input in this area as well): 

![pbta-disease-types](https://user-images.githubusercontent.com/43576623/62727984-ad25d600-b9e8-11e9-8a8a-077ff869c3c1.png)

**Comment by jharenza** [comment]
Hi @cbethell - we actually have a graphic designer on staff that can handle this brain region figure. I think this is a good start, but we are missing major tumor types, eg medulloblastoma, gliomas will have to be narrowed further, and some types have other regions not represented (eg DIPG also in brainstem and thalamus), so this may be difficult to represent and maybe we think about how to do this a bit more. I saw a figure from a paper I liked where they had a b&w brain and the number of tumors per region were in circles, with the circle being proportional in size to the sample N of that region and the color being either molecular subtype or tumor type, so maybe we can iterate around that a bit. (Can't remember the paper offhand, but thought some GBM/HGG subtyping paper).

**Comment by cbethell** [comment]
Hi @jharenza - this seems like a good way to present the major tumor types. I will look for the paper and for other diagrams similar to what you describe above, and prepare the data for use in this manner.

**Comment by jaclyn-taroni** [comment]
@cbethell This issue may be a better spot for discussion around the plots included as part of #40. Can you post what the plots look like here? For the interactive plot, you can probably create a repo of your that includes the HTML file generated in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/40/commits/857fd810e1af16fa20f51df39e579a65de208615 (if I understand correctly) -> turn on [GitHub pages](https://pages.github.com/). Then you can post a link here for easy access.

**Comment by cbethell** [comment]
@jaclyn-taroni per your suggestion, here are the plots included in draft PR #40 :

- `sample-distribution-analyses/plots/distribution_across_cancer_types.pdf`
![distribution_across_cancer_types-1](https://user-images.githubusercontent.com/43576623/62786878-81aaf600-ba91-11e9-9b33-e364cc516792.png)

I modified the above plot to show percentages above each bar instead of the raw count. Thoughts? 

- `sample-distribution-analyses/plots/treemap.pdf`
![treemap-1](https://user-images.githubusercontent.com/43576623/62787817-912b3e80-ba93-11e9-9bf1-101fe708cf14.png)

Above is a treemap plot used to display `broad_histology`, `short_histology`, and `disease_type_new`. `disease_type_new` will be replaced with `molecular_subtype` once determined. What are the thoughts on including the treemap? 


- [`sample-distribution-analyses/plots/histology-pie.html`](https://cbethell.github.io/histology-pie-data/histology-pie.html)
This is the link to the interactive sunburst plot also representing `broad_histology`, `short_histology`, and `disease_type_new` (which again will be changed to `molecular_subtype` once we have that data).

**Comment by cbethell** [comment]
As noted in draft PR #40, the main ideas I would like input on include:
1. Does the multilayer sunburst pie chart meet expectations? (In terms of the presentation and the data representation)
2. Are there any additional data filtering steps that may need to be included for this particular dataset?
3. Are the results/plots suffice for this particular issue? If not, what other forms of tables or plots would you like to see included?

**Comment by jaclyn-taroni** [comment]
For me, the treemap is less valuable as a static image. That is not to say that an interactive treemap would be better than the interactive pie chart you linked above.

**Comment by cbethell** [comment]
> For me, the treemap is less valuable as a static image. That is not to say that an interactive treemap would be better than the interactive pie chart you linked above.

Good point. I can make the treemap interactive and we can decide how useful it may be from there.

**Comment by cbethell** [comment]
> For me, the treemap is less valuable as a static image. That is not to say that an interactive treemap would be better than the interactive pie chart you linked above.

Find the link to the interactive treemap [here](https://cbethell.github.io/histology-pie-data/histology-treemap.html). 

Its value still does not appear to exceed that of the interactive pie chart, however, what are the thoughts on including it now?
